tests/readyflag/tst_readyflag_local.qml

```diff
@@ -0,0 +1,30 @@
+import QtTest 1.2
+import tests.readyflag  1.0
+TestCase {
+    IReadyFlagInterface {
+        id: api
+    }
+    function test_readyInt() {
+        verify(!api.readyFlags.intProperty)
+        api.intProperty = 10;
+        verify(api.readyFlags.intProperty)
+        compare(api.intProperty,10);
+    }
+    function test_readyString() {
+
+        verify(!api.readyFlags.strProperty)
+        api.strProperty = "hello";
+        verify(api.readyFlags.strProperty,true);
+        compare(api.strProperty,"hello");
+    }
+    function test_readyCombo() {
+        verify(!api.readyFlags.comboData)
+        api.comboData.iData = 100;
+        api.comboData.sData = "test";
```

Comment:
this is redundant. maybe a better test would be to reset the ready flag again (to false) and then set sData and see if it is true again - don't know, whether that works, though.
And generally, what about a test were the readyflag changes in the implementation after a function is called?

Reply:
seems its not possible to reset , when property has some value set and its readyflag cannot be reset